src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -197,9 +198,9 @@ void AppendOrdering(IRestRequest request, string orderBy, bool ascending)
             request.AddParameter("direction", ascending ? "asc" : "desc", ParameterType.GetOrPost);
         }
 
-        void AppendPaging(IRestRequest request, int pageIndex)
+        void AppendPaging(IRestRequest request, int pageSize)
```

Comment:
is there anywhere where we change the page size? If it is a constant already and can't be changed, we can also remove the page sizes.